website/docs/support/known-issues.md

```diff
@@ -43,3 +43,7 @@ Below is a list of known Corso issues and limitations:
   updated the locale since the backup was created.
 
 * In-place Exchange contacts restore will merge items in folders named "Contacts" or "contacts" into the default folder.
+
+* Sharepoint list item "attachments" aren't backed up, restored or exported.
+  Graph API doesn't currently provide attachment data for Lists or List Items.
+  An [issue](https://github.com/alcionai/corso/issues/5085) has been created to track the progress..
```

Comment:
We don't need to mention particular issues here, as those are for internal tracking.

```suggestion
```